pkg/lib/cockpit-components-select.jsx

```diff
@@ -92,13 +92,15 @@ Select.propTypes = {
  * Expected properties:
  *  - data (required), will be passed to the select's onChange callback
  *  - disabled (optional): whether or not the entry is disabled.
+ *  - title (optional): acts as a tooltip to the option entry
```

Comment:
This collides with (almost but not quite) the same change in PR #12393. But fine to land this PR first, and rebase the other one.